backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/attribute/impl/HUAttributesBL.java

```diff
@@ -170,4 +178,32 @@ public boolean isAutomaticallySetBestBeforeDate()
 	{
 		return sysConfigBL.getBooleanValue("de.metas.handlingunits.attributes.AutomaticallySetBestBeforeDate", false);
 	}
+
+	@Override
+	public void adjustStorageFromNetWeight(final IAttributeSet attributeSet)
```

Comment:
rename it to `adjustPlanningHUStorageFromNetWeight`.
consider moving it to `IHUReceiptScheduleBL` because atm it's receipt schedule related.